packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/markers_test.dart

```diff
@@ -57,7 +58,12 @@ void main() {
       marker = controller.markers[const MarkerId('1')]?.marker;
       expect(marker, isNotNull);
       expect(marker!.draggable, isFalse);
-      expect(marker.position, isNull);
+
+      // By default, markers fall in LatLng(0, 0)
+      position = marker.position;
```

Comment:
This is quite the intricate dance to fool the linter into thinking this could be nullable. I'd recommend excluding the `expect(... isNotNull)`, or even excluding this behavior from the test spec since it seems to be an artifact of how the API has evolved rather than a design intention.